octobot_trading/enums.py

```diff
@@ -328,15 +363,18 @@ class ExchangeConstantsOrderColumns(enum.Enum):
     SYMBOL = "symbol"
     QUANTITY_CURRENCY = "quantity_currency"
     TYPE = "type"
+    OCTOBOT_ORDER_TYPE = "octobot_order_type"
     SIDE = "side"
     PRICE = "price"
     AMOUNT = "amount"
     COST = "cost"
     AVERAGE = "average"
     FILLED = "filled"
+    FILLED_PRICE = "filled_price"
     REMAINING = "remaining"
     STATUS = "status"
     FEE = "fee"
+    FEES = "fees"
```

Comment:
Should we use `fee` or `fees`? I don't think it's a good idea to have both

Reply:
now its trying to get fee first and if its missing it tries to use fees.
the parsed key will always be fee

Reply:
I don't think we should have both, so if there is always a fee I don't think we should add `fees`.